tests/decap/test_subnet_decap.py

```diff
@@ -0,0 +1,226 @@
+import pytest
+import logging
+import json
+import time
+import random
+from collections import defaultdict
+
+import ptf.packet as packet
+import ptf.testutils as testutils
+from ptf.mask import Mask
+from tests.common.dualtor.dual_tor_utils import rand_selected_interface     # noqa F401
+from tests.common.fixtures.ptfhost_utils import skip_traffic_test           # noqa F401
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py       # noqa F401
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0', 't1')
+]
+
+DECAP_IPINIP_SUBNET_CONFIG_TEMPLATE = "decap/template/decap_ipinip_subnet_config.j2"
+DECAP_IPINIP_SUBNET_CONFIG_JSON = "decap_ipinip_subnet_config.json"
+DECAP_IPINIP_SUBNET_DEL_TEMPLATE = "decap/template/decap_ipinip_subnet_delete.j2"
+DECAP_IPINIP_SUBNET_DEL_JSON = "decap_ipinip_subnet_delete.json"
+
+SUBNET_DECAP_SRC_IP_V4 = "20.20.20.0/24"
+SUBNET_DECAP_SRC_IP_V6 = "fc01::/120"
+OUTER_DST_IP_V4 = "192.168.0.10"
+OUTER_DST_IP_V6 = "fc02:1000::10"
+
+
+@pytest.fixture(scope='module', autouse=True)
+def prepare_subnet_decap_config(rand_selected_dut):
+    logger.info("Prepare subnet decap config")
+    rand_selected_dut.shell('sonic-db-cli CONFIG_DB hset "SUBNET_DECAP|subnet_type" \
+                            "status" "enable" "src_ip" "{}" "src_ip_v6" "{}"'
+                            .format(SUBNET_DECAP_SRC_IP_V4, SUBNET_DECAP_SRC_IP_V6))
+    rand_selected_dut.shell('sudo config save -y')
+    rand_selected_dut.shell('sudo config reload -y')
```

Comment:
better to use [`config_reload` ](https://github.com/sonic-net/sonic-mgmt/blob/e4d40d26a9ae87b88ee5275d9e3d935bf1208af1/tests/common/config_reload.py#L115)

Reply:
Fixed